examples/simpleserver/cmd.go

```diff
@@ -24,7 +25,7 @@ func main() {
 
 func mainWithArgs(ctx context.Context, args []string, logger golog.Logger) error {
 	arm1Name := arm.Named("arm1")
-	arm1, err := fake.NewArm(config.Component{Name: arm1Name.Name}, logger)
+	arm1, err := fake.NewArm(&inject.Robot{}, config.Component{Name: arm1Name.Name}, logger)
```

Comment:
did you test this?  I dont think this will work and wont be run as a part of the test suite

Reply:
did not test of my end.
good catch - will have to update.

Reply:
tested on my end.
The use of `&inject.Robot{}` does not result in any errors.

Reply:
this my terminal output. unsure how to further validate expected behavior.

<img width="1197" alt="Screen Shot 2023-03-08 at 12 05 48 PM" src="https://user-images.githubusercontent.com/46872531/223780717-8c28ffdc-5e14-4083-86d0-2dde5ee057db.png">

Reply:
I think this is probably fine